Azael/app.js

```diff
@@ -24,8 +37,43 @@
 // row 1
 // ```
 
+let counter = 0;
```

Comment:
Task 2 ==> The output starts with row 1 instead of row 0 
Therefore you can set let counter  in line 40 to -1 instead of 0 then you can get row 0, row 1, row2 ....